Cargo.toml

```
@@ -18,6 +18,7 @@ trace = ["dep:opentelemetry"]
 # keep this list sorted!
 [dependencies]
 base64 = "0.22"
+bon = "3.6.4"
```

Nit: We try to keep versions to only the semver significant digits in the toml. This prevents folks newer to rust from mistaking it for pinned versions
```suggestion
bon = "3"
```